prow/gerrit/adapter/adapter.go

```diff
@@ -293,19 +297,28 @@ func (c *Controller) ProcessChange(instance string, change client.ChangeInfo) er
 			logger.WithError(err).Errorf("fail to create prowjob %v", pj)
 		} else {
 			logger.Infof("Triggered Prowjob %s", jSpec.spec.Job)
-			triggeredJobs = append(triggeredJobs, jSpec.spec.Job)
+			triggeredJobs = append(triggeredJobs, triggeredJob{
+				name:   jSpec.spec.Job,
+				report: jSpec.spec.Report,
+			})
 		}
 	}
 
+	var report bool
 	if len(triggeredJobs) > 0 {
-		// comment back to gerrit
-		message := fmt.Sprintf("Triggered %d prow jobs:", len(triggeredJobs))
+		// comment back to gerrit if Report is set for any of the jobs
+		message := fmt.Sprintf("Triggered %d prow jobs (the following list only includes jobs which do not suppress reporting):", len(triggeredJobs))
```

Comment:
nit: Something like `(%d suppressed reporting)` would be a little more concise and informative.